resources/integration_test/tests/query.yaml

```
@@ -0,0 +1,197 @@
+    expected_is_auto_increment: [ false, false, false, false, false ]
+    expected_is_case_sensitive: [ false, false, true, false, false ]
+    expected_is_currency: [ false, false, false, false, false ]
+    expected_is_definitely_writable: [ false, false, false, false, false ]
+    expected_is_nullable: [ columnNullable, columnNullable, columnNullable, columnNullable,
+                            columnNullable ]
+    expected_is_read_only: [ true, true, true, true, true ]
+    expected_is_searchable: [ true, true, true, true, true ]
+    expected_is_signed: [ true, false, false, false, true ]
+    expected_is_writable: [ false, false, false, false, false ]
+
+  - description: select_as_confirm_column_label
+    db: integration_test
+    sql: SELECT enrolled AS e, studentid AS s, name AS n from class
+    expected_result:
+      - [ true, John, 10329 ]
+      - [ false, Jane, 342 ]
+      - [ true, Mike, 303 ]
+      - [ false, Mary, 204323 ]
+      - [ false, Pete, 10 ]
+    row_count: 5
+    ordered: false
+    expected_sql_type: [ BIT, LONGVARCHAR, INTEGER ]
+    expected_catalog_name: [ '', '', '' ]
+    expected_column_class_name: [ boolean, java.lang.String, int ]
+    expected_column_label: [ e, n, s ]
+    expected_column_display_size: [ 1, 0, 10 ]
+    expected_precision: [ 1, 0, 10 ]
+    expected_scale: [ 0, 0, 0 ]
+    expected_schema_name: [ '', '', '' ]
+    expected_is_auto_increment: [ false, false, false ]
+    expected_is_case_sensitive: [ false, true, false ]
+    expected_is_currency: [ false, false, false ]
+    expected_is_definitely_writable: [ false, false, false ]
+    expected_is_nullable: [ columnNullable, columnNullable, columnNullable ]
+    expected_is_read_only: [ true, true, true ]
+    expected_is_searchable: [ true, true, true ]
+    expected_is_signed: [ false, false, true ]
+    expected_is_writable: [ false, false, false ]
+
+  - description: inner_join
+    db: integration_test
+    sql: "SELECT class.name, grades.testid, grades.score FROM grades INNER JOIN class ON grades.studentid\
+      = class.studentid"
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: subquery
+    db: integration_test
+    sql: SELECT studentid FROM (SELECT * FROM class WHERE enrolled = TRUE) AS foo
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: count_and_group_by
+    db: integration_test
+    sql: SELECT COUNT(score), testid FROM grades WHERE score > 85 GROUP BY testid
+    ordered: false
+    skip_reason: SQL-658
+
+  - description: driver_does_not_alter_result_set_row_order
+    db: integration_test
+    sql: SELECT grades.score AS score FROM grades ORDER BY score DESC
+    ordered: true
+    skip_reason: SQL-659
+
+  - description: select_star_from_anyof
+    db: integration_test
+    skip_reason: SQL-668
+
+  - description: select_star_from_any
+    db: integration_test
+    sql: SELECT * from any
+    skip_reason: SQL-696
+
+  - description: select_field_from_any
```

```suggestion
  - description: select_polymorphic_field
```
Alternative name suggestion